### PR TITLE
Revised Web Console Guide

### PR DESCRIPTION
**Pull request by RichardHoch**
Rebranding of Web Console from RHAMT to MTA

Preview: http://file.emea.redhat.com/~rhoch/06252020/Console/

### CODE REVIEW

File: docs/topics_5/web-openshift-deploy.adoc [hunk 1]
```diff
@@ -1,28 +1,26 @@
...
-.. Click the **Browse** button and import the desired {WebName} template. These are included in `<MTA_HOME>/openshift/templates`. Once imported, the JSON is visible in the bottom text area.
-. Click the **Create** button to proceed to the next screen.
-. If you would like to save the template as a resource for future projects, check the `Save template` box.
-. Click the **Continue** button to proceed to the next screen.
-. Review the default values provided, adjusting as necessary. For instance, the `web-template-empty-dir-executor.json` file defines 2 CPUs, 4GB of memory, and a 20GB persistent volume.
-. Click the **Create** button to import the template into your project.
-+
+. Click *Import YAML / JSON`* in the upper-right corner of the web console.
+. Fill in the following fields:
+* *Project Name*. Mandatory.
```
Comment (apinnick) [inline docs/topics_5/web-openshift-deploy.adoc:16]: Put a line break between ordered and unordered lists. It's safer that way. If you don't put in line breaks, occasionally the rendering gets weird.

Comment (RichardHoch) [reply docs/topics_5/web-openshift-deploy.adoc:16]: Done. I noticed something that concerned me in the list of files changed. I exchanged the diagrams and screen captures by the same method: using *Save As* for each item. This ensured that each image reference in the files would work -- and the document built fine. But in the list of files, you see that some, but not all, images were recorded as deleted. Is this a problem?

Comment (apinnick) [reply docs/topics_5/web-openshift-deploy.adoc:16]: I don't think you understand the change that I requested. Or I'm not seeing it.

I also don't understand your question

**Comment by apinnick** [comment]
LGTM. Merging

**Pull request merged**